test/test_suite_test.go

```diff
@@ -312,5 +313,14 @@ var _ = AfterEach(func() {
 			return
 		}
 		_ = os.Remove(filepath.Join(testPath, helpers.MonitorLogFileName))
+
+		// Removed all tcpdump files
+		tcpdumpFiles, err := filepath.Glob(fmt.Sprintf("%s/%s-*.log", testPath, helpers.Tcpdump))
```

Comment:
What is the need to do this in the global after-each?

Reply:
tcpdump files are big, so if the test didn't fail we deleted the files to not store it.

Reply:
Got it, thanks - can you update the comment to indicate that?